includes/class-wc-customer-data-store-custom-table.php

```diff
@@ -129,7 +129,7 @@ public function get_total_spent( &$customer ) {
 			update_user_meta( $customer->get_id(), '_money_spent', $spent );
 		}
 
-		return wc_format_decimal( $spent, 2 );
```

Comment:
Any idea why this was flagged here? Presumably to apply formatting at the time of display?

Reply:
actually this value is formatted where it is used, search for get_total_spent and see its PHPDoc, it says `@return float`